code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -49,14 +49,15 @@
 	var/comp_light_color			//The color of that light
 
 
-
+/obj/item/device/modular_computer/New()
+	all_components = list()
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#startupruntime-tradeoffs-with-lists-and-the-hidden-init-proc